unstructured_inference/models/base.py

```diff
@@ -15,17 +15,41 @@
 from unstructured_inference.models.unstructuredmodel import UnstructuredModel
 from unstructured_inference.models.yolox import MODEL_TYPES as YOLOX_MODEL_TYPES
 from unstructured_inference.models.yolox import UnstructuredYoloXModel
+from unstructured_inference.utils import LazyDict
 
 DEFAULT_MODEL = "yolox"
 
 models: Dict[str, UnstructuredModel] = {}
 
-model_class_map: Dict[str, Type[UnstructuredModel]] = {
-    **{name: UnstructuredDetectronModel for name in DETECTRON2_MODEL_TYPES},
-    **{name: UnstructuredDetectronONNXModel for name in DETECTRON2_ONNX_MODEL_TYPES},
-    **{name: UnstructuredYoloXModel for name in YOLOX_MODEL_TYPES},
-    **{name: UnstructuredChipperModel for name in CHIPPER_MODEL_TYPES},
-}
+
+def get_default_model_mappings() -> (
+    Tuple[
+        Dict[str, Type[UnstructuredModel]],
+        Dict[str, Union[dict, LazyDict]],
```

Comment:
If you add:
```python
from __future__ import annotations
```
at the top of the file you can use modern notation for union and tuple types like:
```
tuple[
    dict[str, type[UnstructuredModel]],
    dict[str, dict | LazyDict],
]
```
which does not require the imports from `typing` (for those types) and is consistent with how Python and IDEs display types in messages etc.

Btw, a bare `dict` is not a type. Here maybe it's `dict[str, Any]`.

Reply:
ah; forgot about future... I was back and forth with lint there 😅